frontend/src/components/HiddenParticipantsTile/HiddenParticipantsTile.tsx

```diff
@@ -44,9 +41,6 @@ const HiddenParticipantsTile = ({
         className="border-none"
         sx={{
           '& .MuiAvatar-root': {
-            borderColor: isHovered ? 'rgb(76,80,82)' : 'rgb(60, 64, 67)',
-            transitionProperty: 'border-color',
-            transitionTimingFunction: 'cubic-bezier(0.4, 0, 0.2, 1)',
```

Comment:
seems like we'd want to keep this transition effect unchanged. can you please elaborate a bit on why this change is needed? thanks!